packages/suite/src/views/onboarding/steps/SecurityCheck/SecurityCheck.tsx

```diff
@@ -231,27 +220,29 @@ const SecurityCheckContent = ({
                 <SecurityChecklist items={checklistItems} />
             </Column>
             <Buttons $elevation={elevation}>
-                <StyledSecurityCheckButton variant="tertiary" onClick={toggleView}>
+                <Button variant="tertiary" onClick={toggleView}>
                     <Translation id={secondaryButtonText} />
-                </StyledSecurityCheckButton>
+                </Button>
                 {initialized ? (
-                    <StyledSecurityCheckButton
+                    <Button
                         data-testid="@onboarding/exit-app-button"
                         onClick={handleContinueButtonClick}
                     >
                         <Translation id="TR_YES_CONTINUE" />
-                    </StyledSecurityCheckButton>
+                    </Button>
                 ) : (
-                    <SecurityCheckButtonWithSecondLine
+                    <Button
                         onClick={handleSetupButtonClick}
                         data-testid="@analytics/continue-button"
                     >
-                        <Translation id={primaryButtonTopText} />
-                        <TimeEstimateWrapper>
-                            <Icon size={12} name="clock" color={theme.iconOnPrimary} />
-                            <Translation id="TR_TAKES_N_MINUTES" />
-                        </TimeEstimateWrapper>
-                    </SecurityCheckButtonWithSecondLine>
+                        <Column>
+                            <Translation id={primaryButtonTopText} />
+                            <TimeEstimateWrapper>
+                                <Icon size={12} name="clock" color={theme.iconOnPrimary} />
+                                <Translation id="TR_TAKES_N_MINUTES" />
+                            </TimeEstimateWrapper>
+                        </Column>
+                    </Button>
```

Comment:
How about using `isFullWidth` prop for all buttons here?

Reply:
It works differently, though.

1. Doesn't work well with `flex-wrap`
2. When only one of the buttons `isFullWidth`, the other one shrinks:
flex-grow:
![Screenshot 2024-09-05 at 17 44 37](https://github.com/user-attachments/assets/a73d3065-e606-420b-bf64-f6a28906c4ff)
width:
![Screenshot 2024-09-05 at 17 37 16](https://github.com/user-attachments/assets/6a666508-89fc-4ae7-b312-03ed56362c8b)
3. When each of the `isFullWidth`, they grow to the same width instead of growing proportionally:
flex-grow:
![Screenshot 2024-09-05 at 17 46 01](https://github.com/user-attachments/assets/0787fa00-5729-4edb-a7ab-26d3acf88e14)
width:
![Screenshot 2024-09-05 at 17 45 21](https://github.com/user-attachments/assets/638541ec-ffc5-47db-b83b-7c9e8043bfb0)